examples/vue3-vite-boilerplate/src/App.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <div>
+    <a href="https://vitejs.dev" target="_blank">
+      <img src="/vite.svg" class="logo" alt="Vite logo" />
+    </a>
+    <a href="https://vuejs.org/" target="_blank">
+      <img src="./assets/vue.svg" class="logo vue" alt="Vue logo" />
+    </a>
+  </div>
+  <scale-button>Hello Scale</scale-button>
+  <span> hello </span>
+</template>
```

Comment:
If we could make this look like this, it'd be great https://github.com/telekom/scale/blob/main/examples/angular-boilerplate/src/app/app.component.html

Reply:
Updated this, I also noticed that the vanilla boilerplate wasn't rendering the scale button correctly anymore so pushed a fix for that too